code/game/objects/items/weapons/traps.dm

```diff
@@ -132,9 +132,18 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
 
 //Using a crowbar allows you to lever the trap open, better success rate
 /obj/item/weapon/beartrap/attackby(obj/item/C, mob/living/user)
-	if (C.has_quality(QUALITY_PRYING))
+	if (C.has_quality(QUALITY_PRYING) && buckled_mob)
 		attempt_release(user, C)
 		return
+	if (deployed && C.w_class > ITEM_SIZE_SMALL)//Occulus Edit Start - Triggering traps with medium items!
```

Comment:
Non-blocker -- this code also allows things like triggering traps with labcoats, etc. with no ill effects. Maybe an item damage component?

Reply:
Eh, anything heavy enough should be able to trip it. I would include a check that forces the item out of your hand, but that might cause issues with cyborg and implants.